onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -1566,7 +1665,10 @@ static constexpr OrtApi ort_api_1_to_3 = {
     &OrtApis::CreateThreadingOptions,
     &OrtApis::ReleaseThreadingOptions,
     &OrtApis::ModelMetadataGetCustomMetadataMapKeys,
-    &OrtApis::AddFreeDimensionOverrideByName};
+    &OrtApis::AddFreeDimensionOverrideByName,
+    &OrtApis::GetAvailableProviders,
+    &OrtApis::ReleaseGetAvailableProviders,
+};
```

Comment:
V1.3 has shipped so this needs a marker saying it's Version 4 similar to the above comments and the new functions need to be after that marker.

Reply:
Done.